modules/api/pkg/handler/v2/provider/gcp.go

```diff
@@ -60,6 +63,61 @@ type gcpSubnetworksNoCredentialReq struct {
 	Network string
 }
 
+type GCPCommonReq struct {
+	ServiceAccount string
+	Credential     string
+}
+
+type GCPProjectCommonReq struct {
+	Request   GCPCommonReq
+	ProjectID string
+}
+
+type GCPVMReq struct {
+	Request GCPCommonReq
+	Zone    string
+}
+
+type GCPProjectVMReq struct {
```

Comment:
The various `Req` structs need to be commented/annotated for swagger to know which API endpoints use it. Here's an example from Nutanix: https://github.com/kubermatic/dashboard/blob/91bdd42fa485303eda47aa1fefbc9335a9502814/modules/api/pkg/handler/v2/provider/nutanix.go#L102-L107

`listProjectNutanixSubnets` is the identifier for the API endpoint which you assign to it in the "big" swagger comment in `routes_v2.go`.